aries/aries_vcx_core/src/wallet/base_wallet/search_filter.rs

```diff
@@ -1,3 +1,9 @@
+#[cfg(feature = "askar_wallet")]
+use aries_askar::entry::TagFilter;
```

Comment:
Fully qualifying the type would avoid this unnecessary feature flag.

Reply:
👍